src/modules/Builder/features/ActivityFlowSettings/ActivityFlowSettings.utils.tsx

```diff
@@ -1,10 +1,11 @@
+import { lazy } from 'react';
 import i18n from 'i18next';
 import { ActivityFlowFormValues } from 'modules/Builder/types';
 import { Svg } from 'shared/components/Svg';
-import { ReportConfigSetting } from 'modules/Builder/features/ReportConfigSetting';
 import { SettingParam } from 'shared/utils';
 
 const { t } = i18n;
+const ReportConfigSetting = lazy(() => import('modules/Builder/features/ReportConfigSetting'));
```

Comment:
maybe it's better to move such imports to normal imports, in this case to the row 6 and make a spacing row after?